hyperstart/api/grpc/hyperstart.proto

```diff
@@ -30,34 +35,39 @@ service HyperstartService {
 	rpc OnlineCPUMem(OnlineCPUMemRequest) returns (google.protobuf.Empty);
 }
 
-message AddContainerRequest {
+message CreateContainerRequest {
 	Container container = 1;
 	Process init = 2;
+	Spec OCI = 3;
+}
+
+message StartContainerRequest {
+	string container_id = 1;
 }
 
-message AddProcessRequest {
-	string container = 1;
+message ExecProcessRequest {
+	string container_id = 1;
```

Comment:
Just a thought about the syntax. Why don't we use `containerID` here instead of `container_id` ?

Reply:
Because `protoc` translate `foo_id` into `fooID` when being told to compile a .proto file to Go.
See hyperstart.pb.go in this PR.

Reply:
Ok thanks